x-pack/plugins/apm/public/components/app/service_node_overview/index.tsx

```diff
@@ -142,7 +142,7 @@ function ServiceNodeOverview({ serviceName }: ServiceNodeOverviewProps) {
   ];
 
   return (
-    <EuiPanel hasShadow={false}>
+    <EuiPanel hasShadow={false} paddingSize="none">
```

Comment:
@formgeist Do you remember if an `EuiPanel` without shadow or padding has any effect? Maybe I can simply remove it.

Reply:
There's no effect, you should be able to remove it.

Reply:
Okay, removing